cloud/test/integration/controllermanager/controllermanager_test.go

```diff
@@ -1452,8 +1452,8 @@ func isOwnedByEdgeApp(ownerReferences []metav1.OwnerReference, edgeApp *appsv1al
 	for _, o := range ownerReferences {
 		if o.APIVersion == appsv1alpha1.GroupVersion.String() &&
 			o.Kind == "EdgeApplication" &&
-			*o.BlockOwnerDeletion == true &&
-			*o.Controller == true &&
+			*o.BlockOwnerDeletion &&
+			*o.Controller &&
```

Comment:
Is it possible that `o.BlockOwnerDeletion` or `o. Controller` is nil, which will cause a null pointer panic?

Reply:
o.BlockOwnerDeletion and o. Controller have specific values，for example 
- apiVersion: apps.kubeedge.io/v1alpha1
    blockOwnerDeletion: true
    controller: true
    kind: EdgeApplication
    name: sample